src/main/java/bisq/core/payment/validation/SpecificAltCoinAddressValidator.java

```diff
@@ -0,0 +1,14 @@
+package bisq.core.payment.validation;
+
+import bisq.core.util.validation.InputValidator;
+
+public interface SpecificAltCoinAddressValidator {
```

Comment:
This name is ugly. I'd rather have it called `AltCoinAddressValidator` but it's already taken by that class that aggregates all alts.